kong/plugins/ddtrace/span.lua

```diff
@@ -74,18 +76,26 @@ function span_methods:set_sampling_priority(sampling_priority)
     self.metrics["_sampling_priority_v1"] = sampling_priority
 end
 
+function span_methods:set_tags(tags)
```

Comment:
Consider making this function _require_ that `type(tags_) == "table"` instead of being a no-op.